src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -79,6 +121,9 @@ private Window AddTopLevelWindow()
                 Height = Dim.Fill()
             };
 
+            win.Accept += () => Accept();
```

Comment:
Is it possible to just do?:

```suggestion
            win.Accept = Accept;
```